gui/wxpython/iscatt/plots.py

```diff
@@ -651,13 +647,9 @@ def __init__(self, ax, pol, empty_pol):
 
         x, y = zip(*self.pol.xy)
 
-        style = self._getPolygonStyle()
-
         self.line = Line2D(x, y, marker="o", markerfacecolor="r", animated=True)
         self.ax.add_line(self.line)
         # self._update_line(pol)
-
-        cid = self.pol.add_callback(self.poly_changed)
```

Comment:
Could you check this one more?

Reply:
I looked at `pol` earlier and it didnt have any function def for this. However I did suspect `add_callback` might be reqd to handle property change. Do I check the function call and just remove `cid` in that case?

Reply:
Yes, just remove the `cid`. In these cases it's helpful to look at how this is used to figure out what `pol` actually is:
https://matplotlib.org/stable/api/_as_gen/matplotlib.artist.Artist.add_callback.html